LICENSE

```diff
@@ -199,4 +198,29 @@
    distributed under the License is distributed on an "AS IS" BASIS,
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
-   limitations under the License.
\ No newline at end of file
+   limitations under the License.
+
+------------------------------------------------------------------------------------
+This product bundles various third-party components under other open source licenses.
+This section summarizes those components and their licenses. See "licenses/" for text
+of these licenses.
+
+
+MIT License
+-----------
+pinot-controller/src/main/resources/*/js/lib/codemirror/*
+pinot-controller/src/main/resources/*/js/lib/foundation/*
+pinot-controller/src/main/resources/*/js/lib/angular*
+pinot-controller/src/main/resources/*/js/lib/underscore*
+pinot-controller/src/main/resources/*/js/lib/jquery-2.1.3.min.js
+pinot-controller/src/main/resources/*/js/lib/beautify.js
+pinot-controller/src/main/resources/*/js/lib/handlebars.js
+pinot-controller/src/main/resources/*/css/lib/codemirror*
+pinot-controller/src/main/resources/*/css/lib/foundation*
+pinot-controller/src/main/resources/*/css/lib/normalize.css
```

Comment:
since I'm not too familiar with this - are we saying these js css files are checked into the git repo?

Reply:
Those files are the part of the git repo (they are used for some UI components. e.g. pinot query console). After checking in this, I will file a separate PR for handling the build to include LICENSE-binary for the binary release.